libraries/AP_Winch/AP_Winch_Daiwa.h

```diff
@@ -0,0 +1,96 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <AP_Winch/AP_Winch_Backend.h>
+#include <SRV_Channel/SRV_Channel.h>
+
+class AP_Winch_Daiwa : public AP_Winch_Backend {
+public:
+
+    AP_Winch_Daiwa(struct AP_Winch::Backend_Config &_config) :
+        AP_Winch_Backend(_config) { }
```

Comment:
```suggestion
    using AP_Winch_Backend::AP_Winch_Backend;
```

Reply:
simplified the daiwa winch constructor, thanks